docs/guides/vault-as-secret-store.md

```diff
@@ -149,6 +149,7 @@ cat << EOF > values.yaml
 args:
 - --enable-external-secret-stores
 customAnnotations:
+  vault.hashicorp.com/agent-inject: \"true\"
```

Comment:
nitpick: do we need to escape the `"` here? Just looking to stay consistent with other annotations below 🙂

Reply:
Very good point. I copied and pasted from the example a few lines below where it was escaped because it was part of an `echo ""`.

Fixed!